extensions/hibernate-orm/runtime/src/main/java/io/quarkus/hibernate/orm/runtime/boot/FastBootMetadataBuilder.java

```diff
@@ -264,8 +266,9 @@ private MergedSettings mergeSettings(QuarkusPersistenceUnitDefinition puDefiniti
         }
 
         //Enable the new Enhanced Proxies capability (unless it was specifically disabled):
-        if (!cfg.containsKey(AvailableSettings.ALLOW_ENHANCEMENT_AS_PROXY)) {
-            cfg.put(AvailableSettings.ALLOW_ENHANCEMENT_AS_PROXY, Boolean.TRUE.toString());
+        final String legacyALLOW_ENHANCEMENT_AS_PROXY = "hibernate.bytecode.allow_enhancement_as_proxy";
```

Comment:
```suggestion
        //FIXME this property is meaningless in Hibernate ORM >6 (and the constant was removed)
        final String legacyALLOW_ENHANCEMENT_AS_PROXY = "hibernate.bytecode.allow_enhancement_as_proxy";
```

Reply:
good point, thanks

Reply:
I suggested adding a comment, not removing one.

But ok, let's ignore that if it's simpler.